style.md

```diff
@@ -1842,6 +1843,46 @@ func run() error {
 </td></tr>
 </tbody></table>
 
+### Use field tags in marshalled structs
+Any struct field that is marshalled into JSON, YAML or other formats, must have relevant tag.
```

Comment:
nit: move the last comma, add a "the":

> Any struct field that is marshalled into JSON, YAML, or other formats must have the relevant tag.

Alternatively, we should probably clarify "formats that support tag-based field naming" (not all do):

```suggestion
Any struct field that is marshaled into JSON, YAML, or other formats that support tag-based field naming should be annotated with the relevant tag.
```